visual-js/visual-nightwatch/integration-tests/nightwatch/tests/saucedemo.ts

```diff
@@ -2,7 +2,9 @@ import { NightwatchTests } from 'nightwatch';
 
 const home: NightwatchTests = {
   'Check Login Page': () => {
-    browser.url('https://saucedemo.com').sauceVisualCheck('Login page');
+    browser.url('https://saucedemo.com').sauceVisualCheck('Login page', {
```

Comment:
This should probably be in a separate test so we can keep the default functionality as is?

Reply:
I Agree.

Ideally we should be able to accept baseline for both situations and ensure this is always equal.

Reply:
moved it to a separate test